src/styles/variables.ts

```diff
@@ -229,4 +229,5 @@ export default {
     mushroomTopHatWidth: 138,
     mushroomTopHatHeight: 128,
     bankButtonMargin: 23,
+    maxAutoGrowHeight: 115,
```

Comment:
```suggestion
    textInputAutoGrowMaxHeight: 115,
```
I think better to include a specified part in the variable name, WDYT?

Reply:
Agree, updated.